appveyor.yml

```diff
@@ -21,8 +21,8 @@ init:
 
 install:
   # Install edm, needed so we can quickly install numpy.
-  - ps: Start-FileDownload "https://package-data.enthought.com/edm/win_x86_64/1.9/edm_1.9.1_x86_64.msi"
-  - start /wait msiexec /a edm_1.9.1_x86_64.msi /qn /log install.log TARGETDIR=c:\
+  - ps: Start-FileDownload "https://package-data.enthought.com/edm/win_x86_64/2.0/edm_cli_2.0.0_x86_64.msi"
+  - start /wait msiexec /a /I edm_cli_2.0.0_x86_64.msi /qn /log install.log TARGETDIR=c:\Enthought\edm
```

Comment:
We needed to change `TARGETDIR` for the other ETS packages. Do we need that change here?

Reply:
ack. yes. we do need to make the change. making the change.